pkg/agent/util/net.go

```diff
@@ -169,3 +169,102 @@ func GetIPWithFamily(ips []net.IP, addrFamily uint8) (net.IP, error) {
 		return nil, errors.New("no IP found with IPv4 AddressFamily")
 	}
 }
+
+// GetAllNodeIPs gets all Node IP addresses (not including IPv6 link local address).
```

Comment:
Do you mean IPv4 link local address is included?

Reply:
IPv6 link local is special, its prefix is `fe80::/64`. It can be only routed. I think it should be as NodePort IP. For IPv6 link local, is there a special range for this?